components/ILIAS/Contact/PRIVACY.md

```diff
@@ -0,0 +1,87 @@
+- Initially the options are restricted to requesting a contact.
+- The requesting user can cancel the request.
+- The requested user has the option to actively ignore or confirm the request.
+- Both contacts can unlink the contact without any further consent.
+- Each party can detect a change of the contact relation within its own contact options. The only exception to this
+  is the active ignoring of a request. The requesting user can not detect if his request was ignored.
+- A list of all contacts is shown as a selection when a user adds members to a newly created mailing list.
+- The "Look up Users" functionality within the composition of a mail can provide a list of contacts if such matches with
+  the search term exist.
+- Furthermore, a list of user accounts with membership assignment is shown by clicking the "List Members" action for an
+  objects within the "My Courses/My Groups" view, accessible directly from the "Compose" view of "Mail"
+  component, or by clicking the "Contacts" link item in the main bar. For each account, the username is presented.
+- Additionally, full names are presented according to the "Profile and Privacy > Publish Profile > Visible for logged-in
+  Users" setting. The "Contact Status" is shown as well.
+- The "Mail to Members" view of membership-enabled object types, which is also located in the "Contact" component,
+  provides a list of participating user accounts. For each account, the username is presented and additionally, the full
+  name, if published.
+  - This information does not show any information about contact relations.
+- Contacts are referenced within the mail service in the sub tab "Contacts".
+- A contact relation cannot be perceived within the user interface by any user other than the contacts themselves.
+
+## Data being deleted
+
+- If a contact request is canceled, the stored entry is removed without any traceable footprint.
+- If a contact request is ignored, the flagged entry in the database will be preserved to prevent a repeated contact
+  request from the same initiator.
+- If a contact is unlinked by one of the contacts, the stored entry is removed without any traceable footprint.
+- Any user relations that originate from the contact are not affected by its deletion. This includes preservation in
+  mailing lists and participation within chatrooms.
+- If a user is deleted, its contacts, send and received contact requests, mailing lists and mailing list participations
+  are removed without any traceable footprint.
+
+## Data being exported
+
+- The component does not provide any export option itself.
+- A contact request can trigger a notification, which may be displayed as a "Toast" to the requested user. The 
+  notification contains the users presentation name (depending on his visibility settings) and the users intention of
+  the contact request.
+  - The notification is handled by the "Notification" component, is self persistent, and therefore not depending on the
```

Comment:
Is it a kind of integrated service? If so, please add a section "Integrated services"/"Services being used" (here are some examples: https://github.com/ILIAS-eLearning/ILIAS/blob/trunk/components/ILIAS/ItemGroup/PRIVACY.md or https://github.com/ILIAS-eLearning/ILIAS/blob/trunk/components/ILIAS/File/PRIVACY.md ) and put this information in this new section. 
If it's not, please ignore this comment.
Some explanation: When I asked you to write privacy.md, the section "Integrated Services" was missing from the"How to write a privacy.md" guide, or did not exist as a separate section. We are in the process of revising the guide and adding this section. We apologize for any inconvenience caused :)

Reply:
This is more a "Used by services" than "Services being used".

At the end, the notification is a completely independent persistent layer on which the contact has no effect retroactively.

It is like the invitation mail to the course service.

Reply:
okay, got it. Thank you for explaining!